test/browser/msw-api/integrity-check.test.ts

```diff
@@ -1,18 +1,19 @@
-import * as fs from 'fs'
-import * as path from 'path'
+import fs from 'node:fs'
 import { test, expect } from '../playwright.extend'
 import copyServiceWorker from '../../../config/copyServiceWorker'
-import { version } from '../../../package.json'
-
-const { SERVICE_WORKER_SOURCE_PATH } = require('../../../config/constants')
+import packageJson from '../../../package.json' assert { type: 'json' }
```

Comment:
Wait wait wait. Dragons here?

AFAIK, import assertions were removed in Node.js v22.0.0, while import attributes became available in Node.js v18.20.5, v20.18.3, and v22.12.0.